server.c

```diff
@@ -18,11 +18,34 @@
 #include "task.h"
 #include "sha1.h"
 
+#define TASK_LOG_FILE "task_log.txt"
+#define MAX_DIFFICULTY 32
+
 static char current_block[MAX_BLOCK_LEN];
 static uint32_t current_difficulty = 0x0000FFF;
 
```

Comment:
At the beginning of the task_log.txt file, it might be great to add a line containing the names of each number below.
eg, "block \ difficulty \ nounce \ username \ current_time"